sdk/search/search/src/generated/data/models/index.ts

```diff
@@ -470,7 +483,7 @@ export interface RequestOptions {
   /**
    * The tracking ID sent with the request to help with debugging.
    */
-  clientRequestId?: string;
+  xMsClientRequestId?: string;
```

Comment:
Not a great name... LMK if I need to fix this in the Swagger somehow.

Reply:
We never expose this field to consumers since OperationOptions already supports setting the request id via a request policy.